labscript_devices/SpinnakerCamera/__init__.py

```diff
@@ -0,0 +1,4 @@
+
+import sys
+if sys.version_info < (3,6):
+    raise RuntimeError("Spinnaker strongly prefers Python 3.6+")
```

Comment:
You can get rid of this check since the new packages will require Python 3.6 at minimum anyway.